tests/conftest.py

```diff
@@ -237,8 +237,9 @@ def mock_model_available(mocker):
 
 
 @pytest.fixture(autouse=True, scope="function")
-def mock_initizalize_client(mocker):
-    mocker.patch.object(LlmApiHandler, "initizalize_client")
+def mock_initizalize_client(mocker, request):
```

Comment:
Typo is still here :upside_down_face:

Reply:
I've really got to make a mentat github PR reviewer.